library/src/main/java/com/joom/grip/io/DirectoryFileSink.kt

```diff
@@ -16,9 +16,14 @@
 
 package com.joom.grip.io
 
+import com.joom.grip.commons.toAbsoluteNormalized
 import java.io.File
+import java.nio.file.Path
+
+class DirectoryFileSink(directoryPath: Path) : FileSink {
+
+  private val directory: File = directoryPath.toAbsoluteNormalized().toFile()
```

Comment:
I'm not sure you really need it here. The only reason of normalization in `FileRegistry` is to make path comparison stable to use paths as keys.

Reply:
yes,  it was made for consistency(like working with normalised paths everywhere), rolled back